src/accounts/fund.rs

```diff
@@ -2,11 +2,11 @@ use serde::{Deserialize, Serialize};
 use serum_common::pack::*;
 use solana_client_gen::solana_sdk::pubkey::Pubkey;
 
-#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq)]
 pub enum FundType {
   /// similar to a gofundme
   FundMe,
-  // Raise,
+  PublicRaise,
```

Comment:
Why do you have this in two areas? One here and the other in src/types

Reply:
types.rs was moved here.